tools/refract-filter.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import argparse
+import sys
+import yaml
```

Comment:
Oh yeah, true. I need to setup proper python environment in my emacs to have all these conventions automated